msteams-platform/concepts/build-and-test/deep-links.md

```diff
@@ -425,7 +425,46 @@ else { /* handle case where capability isn't supported */ }
 
 ```
 
-#### Generate a deep link to a call
+## Generate a deep link to share content to stage in meetings
+
+You can also generate a deep link to [share the app to stage](~/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md#share-entire-app-to-stage) and start or join a meeting.
+
+When deep link is selected in an app and user is already in a meeting, the app is shared to stage, and a permission pop-up window appears. Users can grant permissions for the participants such as co-editing a document or collaborating with an app.
```

Comment:
Is there a screenshot, or a doc page that describes this permission popup window in more detail? It's hard to get a good mental picture of this flow from the text alone.

Is this specific to the deeplinking flow? I wonder if the full description belongs in a different page, and we link to that from here.

Reply:
Added the screenshot of permission pop-up window--"co-edit document".